addPost.php

```diff
@@ -0,0 +1,101 @@
+<?php
+    require_once 'src/Models/BlogModel.php';
+    require_once 'src/connectToDb.php';
+
+    session_start();
+    $db = connectToDb();
```

Comment:
do this under the if statement. no point connecting to the DB if your just going to immediately redirect